src/main/java/org/jabref/gui/mergeentries/newmergedialog/AuthorsFieldRowView.java

```diff
@@ -0,0 +1,30 @@
+package org.jabref.gui.mergeentries.newmergedialog;
+
+import org.jabref.gui.mergeentries.newmergedialog.cell.sidebuttons.WarningButton;
+import org.jabref.gui.mergeentries.newmergedialog.fieldsmerger.FieldMergerFactory;
+import org.jabref.logic.importer.AuthorListParser;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+public class AuthorsFieldRowView extends FieldRowView {
+    private final AuthorList leftEntryAuthors;
+    private final AuthorList rightEntryAuthors;
+
+    public AuthorsFieldRowView(BibEntry leftEntry, BibEntry rightEntry, BibEntry mergedEntry, FieldMergerFactory fieldMergerFactory, int rowIndex) {
+        super(StandardField.AUTHOR, leftEntry, rightEntry, mergedEntry, fieldMergerFactory, rowIndex);
+        var authorsParser = new AuthorListParser();
+        leftEntryAuthors = authorsParser.parse(viewModel.getLeftFieldValue());
+        rightEntryAuthors = authorsParser.parse(viewModel.getRightFieldValue());
+
+        if (leftEntryAuthors.equals(rightEntryAuthors)) {
+            showAuthorsAreTheSameWarning();
+        }
+    }
+
+    private void showAuthorsAreTheSameWarning() {
+        WarningButton warningButton = new WarningButton(Localization.lang("The authors are the same, but the fields are formatted differently."));
```

Comment:
@ThiloteE Do you have a good text for that here? Maybe "The authors are the same. The field content, however, differs syntactically."

Reply:
>  "The authors are the same. The field content, however, differs syntactically."

sounds good and is correct I think, even though I had to look up "syntactically" for its explicit meaning xD

hmmm maybe  "The authors are the same. However, the order of field content differs"? - But this may not be completely accurate: The order of field content in relation to what?